package.json

```diff
@@ -30,7 +30,7 @@
 		"@wordpress/icons": "^6.1.1",
 		"@wordpress/jest-preset-default": "^7.1.3",
 		"@wordpress/prettier-config": "^1.1.1",
-		"@wordpress/scripts": "^19.2.3",
+		"@wordpress/scripts": "^26.19.0",
```

Comment:
It's unclear the potential side-effects of updating this package. The fact that this repository doesn't have CI checks gives low confidence in introducing changes. In any case, I presume that this package is mainly used for the development workflow, so probably won't introduce regressions.

Reply:
Each plugin is manually released separately, so there at least wouldn't be any immediate problems if one of them no longer builds properly.

I ran `yarn everything` to build all the plugins, and didn't see any issues that didn't exist before this change.

Reply:
Great, thank you very much @ajlende for the information and for checking that all plugins can be built 🙇 .